smoke-tests/apps/TraceLogBack/build.gradle.kts

```diff
@@ -13,5 +13,5 @@ dependencies {
     // and duplicate class files produces lots of warning messages on jetty
     exclude("com.microsoft.azure", "applicationinsights-core")
   }
-  implementation("ch.qos.logback:logback-classic:1.2.8")
+  implementation("ch.qos.logback:logback-classic")
```

Comment:
without specifying version number, it will pick up whatever the latest version?

Reply:
it will use the version specified in `dependencyManagement/build.gradle.kts`